### PR TITLE
grains: add to track

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Add `grains` to track

### CODE REVIEW

File: exercises/grains/grains_test.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+@test "Square 1" {
+  #skip
+  run bash grains.sh 1
+
+  [ "$status" -eq 0 ]
+  [ "$output" = "1" ]
+}
+
+@test "Square 2" {
+  #skip
```
Comment (budmc29) [inline exercises/grains/grains_test.sh]: Skips are commented.

File: exercises/grains/grains_test.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  run bash grains.sh -1
+
+  [ "$status" -eq 1 ]
+}
+
+@test "Square 65 causes error" {
+  #skip
+  run bash grains.sh 65
+
+  [ "$status" -eq 1 ]
```
Comment (budmc29) [inline exercises/grains/grains_test.sh]: Expects error.

File: exercises/grains/grains_test.sh (hunk 1 above)
Comment (sjwarner-bp) [reply exercises/grains/grains_test.sh]: Oops! 🙂

File: exercises/grains/grains_test.sh (hunk 2 above)
Comment (sjwarner-bp) [reply exercises/grains/grains_test.sh]: Would you agree the error should be tested by something like:

Comment (budmc29) [reply exercises/grains/grains_test.sh]: Yes, that looks good.

**Comment by sjwarner-bp** [comment]
Hopefully this is better 🙂

**Comment by budmc29** [comment]
Thank you.

**Pull request merged**